core/templates/pages/library-page/library-page.component.html

```diff
@@ -1,3 +1,58 @@
+<div class="classroom-cards-wrapper" *ngIf="publicClassroomsCount > 0">
+  <div class="classroom-cards-inner-container">
+    <div class="classroom-cards-text-container">
+      <h1>{{ 'I18N_COMMUNITY_LIBRARY_PAGE_CLASSROOMS_HINT_TEXT' | translate }}</h1>
+      <p>{{ 'I18N_BROWSE_LESSONS_TEXT' | translate }}</p>
+    </div>
+    <div class="individual-classroom-cards" *ngIf="publicClassroomsCount >= 1 && publicClassroomsCount <= 2">
```

Comment:
"publicClassroomsCount >= 1" can be deleted, right?

Reply:
ah yes we can delete this. Done.